### DISCUSSION TITLE
URL string as default request name

### DISCUSSION THREAD

**Discussion by elgs** [Kong/insomnia#4166] [Ideas]
Everything is great with the only annoying thing that each time I create a new request, a dialog pops up to ask me the request name. At this moment of time, all I want to do is to paste in the url, and my brain was like blocked to think about the request name. I think it will be a big UX improvement to not ask the request name, because people sometimes hate naming things.

**Comment by elgs** [comment]
I finally churned to paw for this reason.

**Reply by huzaifahj** [reply]
Same here
